docs/application/native/guides/app-management/app-controls.md

```diff
@@ -540,9 +545,9 @@ To run a specific application control with some preconfigured parameters:
        dlog_print(DLOG_ERROR, LOG_TAG, "app_control_set_mime() failed. err = %d", ret);
    ```
 
-   For more information on the common application services and the extra data related to them, see [Common Application Controls](common-appcontrol-n.md).
+   For more information on the common application services, see [Common Application Controls](common-appcontrol-n.md).
 
-3. Add extra data to the `app_control` instance using the `app_control_add_extra_data()` or `app_control_add_extra_data_array()` function. In this example, a message is added as extra data:
+3. Add additional data to the `app_control` instance using `app_control_add_extra_data()` or `app_control_add_extra_data_array()`. In this example, a message is added as an additional data:
```

Comment:
@mijin85cho 'an' is used here because it is referring to 'additional' which is starting with a vowel sound. So it is mandatory to use 'an' where a vowel sound is there. Kindly refer to the source for more information: https://www.ecenglish.com/learnenglish/lessons/vowel-sounds-when-use-a-and-an

Reply:
"data" is plural.

https://dictionary.cambridge.org/ko/%EC%82%AC%EC%A0%84/%EC%98%81%EC%96%B4/data
![image](https://user-images.githubusercontent.com/9958374/59410597-30b1b500-8df4-11e9-94b6-3a21739888b3.png)

Reply:
Hello @rishu-samsung @mijin85cho ,
Again agree that "an" should be removed before additional.

Reply:
@annie-samsung, @mijin85cho: Thanks for giving more clarity on this. I'll request for the change with author on this.

@hjhun: Kindly remove 'an' before 'additional data' in the sentence. Please rephrase the sentence as "In this example, a message is added as additional data:"